src/renderer/components/dialog-bisect.tsx

```diff
@@ -41,9 +41,11 @@ export const BisectDialog = observer(
       this.isEarliestItemDisabled = this.isEarliestItemDisabled.bind(this);
       this.isLatestItemDisabled = this.isLatestItemDisabled.bind(this);
 
+      const allVersions = this.props.appState.versionsToShow;
+
       this.state = {
-        allVersions: this.props.appState.versionsToShow,
-        startIndex: 10,
+        allVersions: allVersions,
```

Comment:
```suggestion
        allVersions,
```

This can be simplified since it has the same name as the key.